### PR TITLE
feat: Add page 404  ✨

### PR DESCRIPTION
**Pull request by LUISEDOCCOR**
## Add page 404
image
----------
image

### CODE REVIEW

File: src/pages/NotFound/NotFound.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { Link } from 'react-router-dom';
+
+export const NotFound = () => {
+  return (
+    <RootLayout>
+      <Background />
+      <main className="relative z-2 w-full max-w-7xl mx-auto px-5 flex flex-col justify-center items-center h-[87vh] text-cWhite font-dmsans gap-16">
+        <article className="space-y-lg text-shadow-md">
+          <h1 className="text-7xl font-semibold">Error 404</h1>
+          <h3 className="text-4xl font-semibold">PÁGINA NO ENCONTRADA</h3>
```
Comment (Zyruks) [inline src/pages/NotFound/NotFound.tsx:13]:
```suggestion
          <h2 className="text-4xl font-semibold">PÁGINA NO ENCONTRADA</h2>
```

Seguimos los `heading`, de `h1` luego va `h2`

File: src/pages/NotFound/NotFound.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { ButtonSize, ROUTE, VARIANT } from '@common';
+import { Background, Button } from '@components';
+import { RootLayout } from '@layouts';
+import { Link } from 'react-router-dom';
+
+export const NotFound = () => {
+  return (
+    <RootLayout>
+      <Background />
+      <main className="relative z-2 w-full max-w-7xl mx-auto px-5 flex flex-col justify-center items-center h-[87vh] text-cWhite font-dmsans gap-16">
```
Comment (Zyruks) [inline src/pages/NotFound/NotFound.tsx:10]:
```suggestion
      <main className="relative z-2 w-full max-w-7xl mx-auto px-5 flex flex-col justify-center items-center h-[87svh] text-cWhite font-dmsans gap-16">
```

Con `svh` también hacemos que se vea bien en mobile

File: src/pages/NotFound/NotFound.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            ¡Hola! Lo sentimos, pero no pudimos encontrar lo que buscabas. Verifica que la dirección URL sea correcta
+          </p>
+        </article>
+        <article className="flex justify-center mt-8">
+          <Button size={ButtonSize.xl} variant={VARIANT.GHOST} onClick={() => {}}>
+            <Link to={ROUTE.home} className="flex gap-2 items-center">
+              <span>IR A INICIO</span>
+              <div className="i-lucide:arrow-up-right  w-8 h-8 bg-gradient-to-rb from-primary-600 to-secondary-500" />
+            </Link>
+          </Button>
```
Comment (Zyruks) [inline src/pages/NotFound/NotFound.tsx:24]:
```suggestion
   
  <Link to={ROUTE.home}>
    <Button size={ButtonSize.xl} variant={VARIANT.GHOST} onClick={() => {}} className="flex gap-2 items-center">
      <span>IR A INICIO</span>
      <div className="i-lucide:arrow-up-right  w-8 h-8 bg-gradient-to-rb from-primary-600 to-secondary-500" />
    </Button>
  </Link>
         
```

Esta no estoy totalmente seguro, deberías revisarla.

Si Link está dentro de `Button` cuando cliqueas y no es el texto no pasa nada.

**Review by Zyruks — CHANGES_REQUESTED** [review]
Looks really good, minor changes and is approve :3

**Comment by Zyruks** [comment]
The CI for Commitlint got a error too,

image


You got a commit that is not using the conventional commit structure

image

You can edit this commit, if you need help just let me know.

**Comment by afordigital** [comment]
Very goood PR and very good review! Agree with Zyruks, minor changes approved and its merged!

**Comment by LUISEDOCCOR** [comment]
Thanks Zyruks! ✨But I need some help to change the name to the commit please 🙌

**Comment by juanpablo-is** [comment]
There are several ways, since it is not a commit so far down, you can do:

`git reset d0bdffda5e319d3cb8ac280ec64160b7216da0f1`
`git commit --amend -m "feat: update routes.ts"`
`git add .`
`git commit -m "feat: add wildcard route for 404 error"`

**Comment by LUISEDOCCOR** [comment]
Thank you so much @juanpablo-is 🙌

**Pull request closed without merging**